src/Models/VisitorsModel.php

```diff
@@ -18,13 +18,18 @@ class VisitorsModel extends BaseModel
      */
     public function countAllVisitors($args = [], $bypassCache = false)
```

Comment:
@Koorosh14  
We already have `countVisitors `method, which we can use for this purpose. Could you please merge these two functions if possible? If it should be separate, please use another name to avoid confusion, like `countDevicesData `.

Reply:
Thank you for the comment. 
Yes, the query (and the return type) were different, that's why I added a new method.

Reply:
Thank you so much! So let's go with the `countDevicesData` then.